src/styles/blocks/header.scss

```diff
@@ -0,0 +1,22 @@
+.header {
+  box-shadow: 0 2px 4px $color-black-shadow;
+
+  &__content {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    max-width: 1200px;
+    margin: 0 auto;
+    padding: 0 50px;
+  }
+
+  &__logo {
+    display: block;
+
+    height: 40px;
+    width: 40px;
```

Comment:
stars have width and heigth = 16 * 16, it's a square, logo have width and heigth 40*40. also a square. mixin with variable?